test/metabase/driver/sql_jdbc/sync/describe_database_test.clj

```diff
@@ -208,3 +208,23 @@
                                   driver/*driver* conn schema (str table-name "_should_not_exist"))))
                      (is (true? (sql-jdbc.sync.interface/have-select-privilege?
                                  driver/*driver* conn schema table-name))))))))))))))
+
+(deftest sync-table-with-backslash-test
```

Comment:
If we're going to do it for all drivers, ideally we would also test on forward-slashes too, since forward slashes were the problem in https://github.com/metabase/metabase/pull/13340